### ISSUE TITLE
Navigation labels (Choose service)

### ISSUE THREAD

**Issue by adriannelee** [cds-snc/notification-planning#577]
Issue group: Your services
Visibility: 2
Evaluation: Partially Supports
Success Criteria: 2.4.6: Headings and Labels
Needs design: Content

### Description
Navigation should be appropriately labelled to identify the type of navigation it contains and distinguish it from other navigation elements on the page. Some ATs allow users to list and navigate to the various navigation elements on a page.

1. The navigation labelled “Language and sign-in” may be misleading as it also appears after the user has already signed in. After logging in, it may be better labelled like “Language and account”. There may also be a generic name that works well in both cases.
2. On the Your services page, the navigation element containing links to the services, is not labelled.

![](https://dl.airtable.com/.attachments/ac7750f3e84de8e0eff051a311aefe3c/7c6a18cc/Capturedecranle2022-03-15a13_39_26.png)

**Comment by amazingphilippe** [comment]
Related finding from testing:

https://airtable.com/appWwAmHwDLtpIyko/tbl38n7ofWYBuezFc/viwBYyaQxOttZ0IAw/recAYwlRcb3Q7daMG?blocks=hide

> GC Notify was difficult to navigate for participants. For a user using a screenreader, the labels weren't descriptive enough (e.g. navigation regions, homepage link). For a screen magnification user, the grey banner on the top nav was too light (but other screen magnification users didn't comment on it)

**Comment by amazingphilippe** [comment]
[Figma for labels](https://www.figma.com/file/r3GzRBjBDWvL2AwCWq1e1W/Navigating-the-site-%7C-Naviguer-le-site?node-id=7800%3A4588)

**Comment by andrewleith** [comment]
Looks good.
